source/lib/ze_lib.cpp

```diff
@@ -74,6 +74,11 @@ namespace ze_lib
             ze_lib::context->zesDdiTable.exchange(&ze_lib::context->initialzesDdiTable);
         }
 
+        // Given zesInit, then zesDrivers needs to be used as the sysmanInstanceDrivers;
+        if (sysmanOnly) {
+            loader::context->sysmanInstanceDrivers = &loader::context->zesDrivers;
```

Comment:
Just verifying, this runs *after* the change in ze_loader.cpp?

Reply:
yes